src/lib/viewers/doc/_docBase.scss

```diff
@@ -336,3 +336,23 @@ $thumbnail-sidebar-width: 226px;
 .bp-content.bp-is-fullscreen .bp-toggle-thumbnails-icon {
     display: none;
 }
+
+.bp-content {
```

Comment:
Maybe we can roll the styles in L336 into this one as well?

Reply:
Yeah, instead of combining L340 and L341, you can move L336 too